docs/table/table_and_dataframes.rst

```diff
@@ -0,0 +1,95 @@
+.. _astropy-table-and-dataframes:
+
+Astropy Table and DataFrames
+============================
+
+`Pandas <https://pandas.pydata.org/>`_ is a popular data manipulation library for Python
+that provides a `~pandas.DataFrame` object which is similar to `astropy.table`. A common
+question is why Astropy does not use `~pandas.DataFrame` as the base table object. The
+answer stems from a number of domain-specific requirements related to astronomical data
+and analysis.
+
+Units and Quantities
+--------------------
+
+Astronomy is a physical science, and the data often have units associated with
+them. The `astropy.table` package natively supports |Quantity| columns, which are a
+powerful way to attach units to array data and perform unit-aware operations. In
+addition, the base `~astropy.table.Column` class holds a ``unit`` attribute as
+metadata to allow tracking of the units of the data for applications not using
+|Quantity|.
+
+Pandas does not provide support for units.
+
+Multi-dimensional and Structured Columns
+----------------------------------------
+
+Astronomers deal with images, spectra, and other multi-dimensional data that are
+commonly stored in a table. An example is a source catalog with an image thumbnail and a
+spectrum for each source. Structured columns are less common, but are useful for storing
+vectorized data like an `~astropy.coordinates.EarthLocation` in a table.
+
+Pandas is not able to natively store multi-dimensional or structured columns.
+
+Lossless representation of FITS and VOTable data via metadata
+-------------------------------------------------------------
+
+The `astropy.table` package is designed to provide lossless representation of FITS and
+VOTable data. This means that when you read a FITS or VOTable file into a table and then
+write it back out, the data will be effectively identical. This is made possible by
+robust support for table and column metadata which allows storing and propagating common
+column information such as the unit, description, and format. For VOTable data, more
+esoteric information like the UCD is maintained.
```

Comment:
can we use another, more neutral adjective here? Or even just drop the adjective
```suggestion information like the UCD is maintained.
```

Reply:
Fixed.